### PR TITLE
chore/401 add new script to update tests

### PR DESCRIPTION
**Pull request by agar23**
**Type:**

The following has been addressed in the PR:

*  There is a related issue? #401 
*  Unit or Functional tests are included in the PR

**Description:**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -171,6 +171,7 @@
     "test:acceptance:server": "concurrently --kill-others \"npm run verdaccio:server\" \"npm run test:acceptance\"",
     "test:e2e": "cross-env BABEL_ENV=test jest --config ./test/jest.config.e2e.js",
     "test": "cross-env NODE_ENV=test BABEL_ENV=test TZ=UTC jest --config ./jest/jest.config.js --maxWorkers 2 --passWithNoTests",
+    "test:snapshopts": "npm run test -- -u",
```
Comment (sergiohgz) [inline package.json]: The script name tells me more to run snapshot tests than update. Also, snapshoPts is misspelled.

Comment (agar23) [reply package.json]: make sense, updated to `test:update-snapshot`

Comment (juanpicado) [reply package.json]: I would rather go test:update and using yarn instead npm

Comment (DanielRuf) [reply package.json]: But we use `npm ...`in other scripts too. If this should be changed to `yarn`, shouldn't it be for all or for more?

Comment (sergiohgz) [reply package.json]: We should migrate all scripts to use one package manager, but that is other issue, let's open a new one and keep this simple :wink:

Comment (DanielRuf) [reply package.json]: Exacctly.

**Pull request merged**